src/PrestaShopBundle/Resources/views/Admin/Module/Includes/card_list.html.twig

```diff
@@ -88,6 +88,11 @@
               ...
             {% endif %}
           {% endblock %}
+          {% block addon_additional_description %}
+            {% if module.attributes.additional_description is defined %}
+              {{ module.attributes.additional_description|raw }}
```

Comment:
Just a question since we use `|raw` here isn't there a risk of injecting malicious code?
@NeOMakinG what do you think?